backend/tests/conftest.py

```diff
@@ -1,9 +1,23 @@
+import os
+
 import pytest
 from fastapi.testclient import TestClient
+from neo4j import GraphDatabase
 
 from editor.api import app
 
 
+# clean database before running tests
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
I would say we want it not at session level but for each test !

I would separate it in two part:

* a session fixture to get the driver
* a utility function to clean db. In most test files, where you use database, you could add a setup method, in autouse which calls this the clean db (and eventually setup some data).